rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -2134,10 +2205,31 @@ mod test {
         state
     }
 
-    /// Add some placeholder wasms with different values so we can test
-    /// that each value is installed into the correct spot
-    /// group_number tells you which group of wasms you are adding so that they
-    /// will have different content and therefore different hashes
+    // Adds section "icp:[public|private] $name$contents" to `wasm`, returning `wasm`'s new hash.
+    fn annotate_wasm_with_metadata(
```

Comment:
This is confusing, b/c it returns a hash instead of a wasm that's annotated.  Could we fix the function name?

Reply:
Note: This test function was there already before this PR.

I'm going to change it here nonetheless, if I get to it.